vpd-tool/include/vpd_tool.hpp

```diff
@@ -39,5 +39,32 @@ class VpdTool
                     const std::string& i_recordName,
                     const std::string& i_keywordName, const bool i_onHardware,
                     const std::string& i_fileToSave = {});
+
+    /**
+     * @brief Write keyword's value.
+     *
+     * API to update VPD keyword's value to the given input path.
+     * If i_onHardware value in true, i_fruPath is considered has hardware path
+     * otherwise it will be considered as DBus object path.
+     *
+     * Regardless of given input path is primary path or secondary path,
+     * internally both paths will get updated, also redundant EEPROM(if any)
+     * path with new keyword's value.
+     *
+     * @param[in] i_vpdPath - DBus object path or EEPROM path.
+     * @param[in] i_recordName - Record name.
+     * @param[in] i_keywordName - Keyword name.
+     * @param[in] i_keywordValue - Keyword value.
+     * @param[in] i_onHardware - True if i_vpdPath is EEPROM path, false
+     * otherwise.
+     * @param[in] i_filePath - File path to take keyword's value.
+     *
+     * @return On success return 0, otherwise return -1.
+     */
+    int writeKeyword(const std::string& i_vpdPath,
+                     const std::string& i_recordName,
+                     const std::string& i_keywordName,
+                     const std::string& i_keywordValue, const bool i_onHardware,
+                     const std::string& i_filePath = {});
```

Comment:
`writeKeyword` API should have `const` and `noexcept` specifiers as it : 

1. does not modify any members of `VpdTool` class and
2. does not throw any exceptions

Reply:
> as VpdTool class doesn't have any member variables'

Agreed that it does not have any members now , but we may add in the future right?

Reply:
we add add things later, when the need arises.